src/renderer/containers/ReplayBrowser/index.tsx

```diff
@@ -161,17 +176,61 @@ export const ReplayBrowser: React.FC = () => {
                     <FileList
                       folderPath={currentFolder}
                       onDelete={deleteFile}
+                      handleAddToList={handleAddToList}
+                      list={list}
                       onSelect={(index: number) => setSelectedItem(index)}
                       onPlay={(index: number) => playSelectedFile(index)}
                       files={filteredFiles}
                       scrollRowItem={scrollRowItem}
                       setScrollRowItem={setScrollRowItem}
                     />
                   )}
+                  {list.length > 0 ? (
+                    <div
+                      css={css`
+                        position: absolute;
+                        bottom: 0px;
+                        right: 0px;
+                        margin-right: 24px;
+                        margin-bottom: 16px;
+                      `}
+                    >
+                      <Button
+                        css={css`
+                          background-color: rgb(220, 219, 220);
+                          color: black;
+                        `}
+                      >
+                        Play All
+                      </Button>
+
+                      <Button
+                        onClick={() => setList([])}
+                        css={css`
+                          background-color: rgb(220, 219, 220);
+                          color: black;
+                          margin-left: 12px;
+                        `}
+                      >
+                        Clear
+                      </Button>
+
+                      <Button
+                        css={css`
+                          background-color: rgb(220, 219, 220);
+                          color: black;
+                          margin-left: 12px;
+                        `}
+                      >
+                        Delete
+                      </Button>
+                    </div>
+                  ) : null}
```

Comment:
Can you move this into a separate component? Call it like `FileSelectionToolbar.tsx` or something inside of `src/renderer/containers/ReplayBrowser`.